flight/Modules/Logging/logging.c

```diff
@@ -339,6 +343,8 @@ static void loggingTask(void *parameters)
 					register_default_profile();
 					break;
 				case LOGGINGSETTINGS_PROFILE_CUSTOM:
+					register_custom_profile();
```

Comment:
Do we really need this?  Or should we just change metadata so that the default profile logs this stuff.

Reply:
This was going to go away at some point, anyway. I have this in the test branch to allow faster logging without hitting the CPU limit.